Sluggo-SwiftUI/Views/Components/OptionalOperator.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  OptionalOperator.swift
+//  Sluggo-SwiftUI
+//
+//  Created by Troy Ebert on 3/10/23.
+//
+
+import Foundation
+import SwiftUI
+
+func ??<T>(lhs: Binding<Optional<T>>, rhs: T) -> Binding<T> {
```

Comment:
What is this used for?

Reply:
This is the ?? operand, used for passing optional generics, I use this specifically for optional ticket fields in text fields